securedrop/sass/source.sass

```diff
@@ -66,9 +66,6 @@ input#show
 #content
```

Comment:
It looks like this `content` id is also used in `lookup.html` on the source interface - what do you think about instead removing the surrounding `<div id="content">` for both `error.html` and `notfound.html`? It looks like that change fixes this and renders well:

![screen shot 2017-07-16 at 12 43 04 pm](https://user-images.githubusercontent.com/7832803/28250728-94f0ed4c-6a24-11e7-8eb4-1bb155421ff6.png)